dir.traversal.targets

```diff
@@ -78,7 +78,7 @@
       </Project>
 
       <!-- Include all projects which have TestTFM from the supported set -->
-      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.TestTFMs)').Contains(';$(FilterToTestTFM);'))">
+      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.Identity)').Contains('.Tests'))">
```

Comment:
As for filtering I think the filtering should be handled by build filtering. While doing a vertical build we should run the tests for any build we do. There isn't going to be the same type of multiplexing on a single test project that we had in the past. We will essentially to the matching build and always run based on the BuildConfiguration provided or defaulted.